config/const.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+return [
+    'role_id' => [
+        'role_id' => 1,
```

Comment:
```suggestion
    'role' => [
        'admin' => 1,
        'user' => 0
     ],
        
```
Dùng config thì đặt enum như này để nó tường mình e nhé
Đọc qua cái là hiểu số 1 đó là gì luôn